source/features/index.tsx

```diff
@@ -78,7 +78,7 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 		return;
 	}
 
-	if (document.title === 'Confirm password') {
+	if (document.title === 'Confirm password' || document.title === 'Confirm access') {
```

Comment:
Where does this appear?

Reply:
This is the password page. It should not be in this PR. I put it in the other PR.

Reply:
"the password page" is 'Confirm password'.

Reply:
I found some pages they changed to "Confirm access"

I found it when i tried adding a collaborator to a repo of mine